docs/chain/0_introduction.md

```diff
@@ -3,4 +3,7 @@ id: introduction
 title: Introduction
 ---
 
-WIP: gentle introduction into the KILT Blockchain!
+This section covers chain specific topics.
+
+* You can find out [how to become a collator](collator.md)
+* [Become a delegator](delegator.md) by following this guide
```

Comment:
It feels either we change one or the other to the following. `How to` or `Become` etc.